### ISSUE TITLE
As a developer I can use a package manager to fetch/update Previews from a feed so I don’t have to do this manually

### ISSUE THREAD

**Issue by jamshedd** [dotnet/core#5445]

**Comment by richlander** [comment]
I don't think it makes sense to move this content to Learn since it isn't intended as the primary way to install packages once .NET 9 goes GA. Release notes are intended for folks that have the motivation and interest to read them. You found them and now have the information. Sorry for not realizing that this was a solution to your problem.

If @adegeo feels differently, that's fine with me.

**Comment by perlun** [comment]
> I don't think it makes sense to move this content to Learn since it isn't intended as the primary way to install packages once .NET 9 goes GA. Release notes are intended for folks that have the motivation and interest to read them. You found them and now have the information. Sorry for not realizing that this was a solution to your problem.

I hear you, **but** my main objection to this is that the release notes _do_ mention a way to install the preview packages for Linux. That "way" is much less optimal for some people (like me) because it means that you get a `dotnet` binary which is "isolated" from all your other .NET SDKs. Good sometimes (because as mentioned elsewhere, the `dotnet` binary from the .NET Preview can cause breakage with existing SDK installations) but the thing I found really annoying about it was that I could no longer run `dotnet test` (with the "isolated preview" binary) for my .NET 8 projects - because the isolated preview is indeed isolated and failed to find the other .NET SDKs. No screenshot/log for this since I unfortunately already removed the files from my system but I'm sure you know what I'm talking about.

With the link from the release notes, I get a .NET 9 Preview which is much more "integrated" with my system, for better and worse.

Short version: both these ways of installing the .NET Preview are valid and deserves to be mentioned. The way the Learn text is written now gives the pretension that "these are (all) the ways you can install the preview" when this is not the case. That's my objection to it.

**Comment by jamshedd** [comment]
Adding @ashnaga

**Comment by ashnaga** [comment]
> I think it would be proper to have a general "Install a .NET Preview" doc page instead of merging this info into the main install articles. Thoughts on that?

Like the idea to keep it as separate doc page as not all ways of install and update will be applicable to previews.

**Comment by perlun** [comment]
Works for me. :+1: Ideally, the page at https://learn.microsoft.com/en-us/dotnet/core/install/linux-debian#install-preview-versions would then link to the new "Install a .NET Preview", which can be maintained separately (and more accurately describe various ways to install the previews on Linux).
